oap-server/server-library/library-server/src/main/java/org/apache/skywalking/oap/server/library/server/grpc/GRPCServer.java

```diff
@@ -90,12 +93,26 @@ public String serverClassify() {
     @Override
     public void initialize() {
         InetSocketAddress address = new InetSocketAddress(host, port);
-
+        ArrayBlockingQueue blockingQueue = new ArrayBlockingQueue(10000);
+        ExecutorService executor = new ThreadPoolExecutor(500, 500, 60,
```

Comment:
All these parameters require configurable, and default values should be reasonable, such as calculated from CPU core number. Such as take a look at `MetricsAggregateWorker#L52`
```JAVA
BulkConsumePool.Creator creator = new BulkConsumePool.Creator(name, BulkConsumePool.Creator.recommendMaxSize() * 2, 20);
```